services/registration/controller/controller.go

```diff
@@ -234,7 +234,7 @@ func GetFilteredUserRegistrations(w http.ResponseWriter, r *http.Request) {
 	user_registrations, err := service.GetFilteredUserRegistrations(parameters)
 
 	if err != nil {
-		panic(errors.UnprocessableError(err.Error()))
+		panic(errors.DatabaseError(err.Error(), "Could not get filtered user registrations.\n"+err.Error()))
```

Comment:
Don't need a newline here and don't need to append `err.Error()` since it will be included in the `RawError` field.